src/libraries/System.Security.Cryptography.Primitives/tests/CryptoStream.cs

```diff
@@ -58,17 +58,6 @@ public static void Roundtrip(int inputBlockSize, int outputBlockSize, bool canTr
                 Assert.False(encryptStream.CanRead);
                 Assert.False(encryptStream.CanSeek);
                 Assert.False(encryptStream.HasFlushedFinalBlock);
-                Assert.Throws<NotSupportedException>(() => encryptStream.SetLength(1));
```

Comment:
Why were these Asserts removed?

Reply:
They're duplicates of what's in the base class.